### DISCUSSION TITLE
Configurable Blue Streak on load/refresh

### DISCUSSION THREAD

**Discussion by xPB12** [grafana/grafana#69924] [Dashboards, Panels & Visualizations]
The latest versions of Grafana added a blue streak when the visualisations are loading/refreshing. Although it is a cool feature and gives realtime feedback, it can be a distraction when the refresh interval is set to anything less than 5s on a dashboard with multiple visualisations.

I would like an option to fall back to the previous way of refreshing with a refresh icon on the visualisations or an option to independently turn off the blue streaks altogether.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82654. Feel free to continue the discussion around this there. Thank you!
